go/libraries/doltcore/sqle/dfunctions/dolt_merge.go

```diff
@@ -102,108 +94,110 @@ func (d DoltMergeFunc) Eval(ctx *sql.Context, row sql.Row) (interface{}, error)
 		return noConflicts, nil
 	}
 
-	ddb, ok := sess.GetDoltDB(ctx, dbName)
-	if !ok {
-		return noConflicts, sql.ErrDatabaseNotFound.New(dbName)
+	branchName := apr.Arg(0)
+
+	mergeSpec, err := sqlMergeSpec(ctx, sess, dbName, apr, branchName)
+	if err != nil {
+		return noConflicts, err
+	}
+	ws, conflicts, err := mergeHelper(ctx, sess, roots, ws, dbName, mergeSpec)
+	if err != nil {
+		return conflicts, err
+	}
+
+	err = sess.SetWorkingSet(ctx, dbName, ws, nil)
+	if err != nil {
+		return conflicts, err
 	}
 
+	return conflicts, nil
+}
+
+func mergeHelper(ctx *sql.Context, sess *dsess.Session, roots doltdb.Roots, ws *doltdb.WorkingSet, dbName string, spec *merge.MergeSpec) (*doltdb.WorkingSet, interface{}, error) {
```

Comment:
mergeIntoWorkingSet

Reply:
I know you didn't create these methods originally, but returning `interface{}` for the two constants `hasConflicts` and `noConflicts` is not great. Define an integer type alias for them and declare the consts in terms of that type.

Reply:
Also add a method doc here, pay attention to the return values